### PR TITLE
Change Header Fixed width. It will crop out on resize.

### PR DESCRIPTION
**Pull request by AjayDoddapaneni**

### CODE REVIEW

File: src/main/frontend/src/components/Header/HeaderView.js [hunk 1]
```diff
@@ -4,10 +4,10 @@ import { withStyles } from '@material-ui/core';
 import nihLogo from '../../assets/header/CTDC_Logo.svg';
 // import icdcLogo from '../../assets/header/icdc_logo_white.svg';
 // import cancergraphic400 from '../../assets/header/Canine400.png';
-import cancergraphic800 from '../../assets/header/easter800.png';
-import cancergraphic1400 from '../../assets/header/easter1400.png';
-import cancergraphic1600 from '../../assets/header/easter1600.png';
-import cancergraphic1800 from '../../assets/header/easter1800.png';
+// import cancergraphic800 from '../../assets/header/easter800.png';
```
Comment (AjayDoddapaneni) [inline src/main/frontend/src/components/Header/HeaderView.js:7]: If you are no longer gonna use this remove the files can you please clean up assets folder too.

File: src/main/frontend/src/components/Header/HeaderView.js [hunk 2]
```diff
@@ -67,37 +67,38 @@ const styles = () => ({
...
-    },
-    '@media (min-width: 2400px)': {
-      backgroundRepeat: 'no-repeat',
-      backgroundSize: '100% 100%',
-      background: `url(${cancergraphic2000})`,
-    },
+    background: `url(${cancergraphic2000})`,
+    overflow: 'hidden',
+    // '@media (min-width: 900px)': {
+    //   backgroundRepeat: 'no-repeat',
```
Comment (AjayDoddapaneni) [inline src/main/frontend/src/components/Header/HeaderView.js:73]: Can you please remove the commented code.

File: src/main/frontend/src/components/Header/HeaderView.js (hunk 1 above)
Comment (iksheth) [reply src/main/frontend/src/components/Header/HeaderView.js:7]: Sure.

File: src/main/frontend/src/components/Header/HeaderView.js (hunk 2 above)
Comment (iksheth) [reply src/main/frontend/src/components/Header/HeaderView.js:73]: Sure.

**Comment by iksheth** [comment]
@AjayDoddapaneni Can you please review changes?

**Pull request merged**